packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,162 @@
+      background-position: center center;
+      background-repeat: no-repeat;
+      border-color: var(--terra-form-checkbox-border-color, none);
+      border-radius: var(--terra-form-checkbox-border-radius, none);
+      border-style: var(--terra-form-checkbox-border-style, none);
+      border-width: var(--terra-form-checkbox-border-width, none);
+      content: '';
+      cursor: pointer;
+      display: inline-block;
+      height: var(--terra-form-checkbox-height, none);
+      left: 0;
+      margin-top: var(--terra-form-checkbox-margin-top, none);
+      position: absolute;
+      top: 0;
+      width: var(--terra-form-checkbox-width, none);
+    }
+
+    /* Custom checkbox in checked state */
+    .native-input:checked + .label-text::before {
+      background-color: var(--terra-form-checkbox-checked-background-color, none);
+      border-color: var(--terra-form-checkbox-checked-border-color, none);
+      border-radius: var(--terra-form-checkbox-checked-border-radius, none);
+      border-style: var(--terra-form-checkbox-checked-border-style, none);
+      border-width: var(--terra-form-checkbox-checked-border-width, none);
+      color: var(--terra-form-checkbox-symbol-color, none);
+      content: var(--terra-form-checkbox-symbol, none);
+      height: var(--terra-form-checkbox-checked-height, none);
+      left: 0;
+      line-height: 0.9;
+      margin-top: var(--terra-form-checkbox-margin-top, none);
+      opacity: 1;
+      position: absolute;
+      text-align: center;
+      top: 0;
+      transform: scale(1);
+      transition: var(--terra-form-checkbox-transition, all 0.2s ease all);
+      vertical-align: middle;
+      visibility: visible;
+      width: var(--terra-form-checkbox-checked-width, none);
+    }
+
+    /* Hide label if isLabelHidden true */
+    .is-hidden {
+      cursor: default;
+      height: 1rem;
+      margin-left: 0;
+      width: 0;
+    }
+
+    /* Disabled native checkbox, label text, and custom checkbox styling */
+    .is-disabled {
+      cursor: default;
+
+      > .label-text {
+        color: var(--terra-form-checkbox-disabled-font-color, #9b9fa1);
+        cursor: default;
+        opacity: var(--terra-form-checkbox-disabled-font-opacity, none);
+      }
+
+      > .label-text::before {
+        background-color: var(--terra-form-checkbox-disabled-background-color, none);
+        background-image: var(--terra-form-checkbox-disabled-background-image, none);
+        border-color: var(--terra-form-checkbox-disabled-border-color, none);
+        cursor: default;
+      }
+    }
+
+    /* Focus custom checkbox styling */
+    .native-input:focus + .label-text::after {
+      [dir=rtl] & {
+        box-shadow: var(--terra-form-checkbox-focus-box-shadow, none);
+      }
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-form-checkbox-focus-box-shadow, none);
+      }
+    }
+  }
+
+  /* Focus style surrounding both custom checkbox and label focus. Utilized in Clinical */
```

Comment:
I'd be careful with comments related to themes. They are proprietary, meaning it's best not to mention their intended use in our comments.

For this comment, I'd just remove the entire comment. The focus class self documents the code.